js/web/package.json

```diff
@@ -1,111 +1,111 @@
-    "start:renderer": "cross-env SKIP_PREFLIGHT_CHECK=true PORT=3002 BROWSER=none craco start",
-    "build:main": "npx tsc --project tsconfig.main.json --watch",
-    "dist:win": "npx webpack-cli build -c webpack.main.config.js && electron-builder -w -c.extraMetadata.main=build/main.js",
-    "dist:mac": "npx webpack-cli build -c webpack.main.config.js && electron-builder -m -c.extraMetadata.main=build/main.js  -c.artifactName=Berty.dmg",
-    "dist:linux": "npx webpack-cli build -c webpack.main.config.js && electron-builder -l -c.extraMetadata.main=build/main.js",
-    "fix:electron": "npx electron-fix start",
-    "start:electron": "concurrently \"npm:start:renderer\" \"npm:build:main\" \"wait-on http://127.0.0.1:3002 && cross-env NODE_ENV=development electronmon .\"",
-    "start:electron:no-daemon": "concurrently \"npm:start:renderer\" \"npm:build:main\" \"wait-on http://127.0.0.1:3002 && cross-env NO_DAEMON=true NODE_ENV=development electronmon .\""
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not op_mini all"
-  ],
-  "homepage": ".",
-  "main": "./build/main.js",
-  "engines": {
-    "node": ">=14"
-  },
-  "resolutions": {
-    "react-error-overlay": "6.0.9"
-  },
-  "build": {
-    "extends": null,
-    "appId": "tech.berty.mac",
-    "productName": "Berty",
-    "files": [
-      "build/**/*",
-      "!build/**/*.map",
-      "!**/node_modules/**"
-    ],
-    "extraFiles": [
-      {
-        "from": "resources/${os}",
-        "to": "bin",
-        "filter": [
-          "**/*"
-        ]
-      }
-    ],
-    "directories": {
-      "buildResources": "build"
-    },
-    "mac": {
-      "target": "dmg",
-      "category": "social-networking"
-    },
-    "win": {
-      "target": "nsis",
-      "icon": "./public/logo512.png"
-    },
-    "linux": {
-      "target": "deb",
-      "category": "Development"
-    }
-  },
-  "devDependencies": {
-    "@babel/core": "^7.16.12",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-flow": "^7.16.7",
-    "@babel/preset-react": "^7.16.7",
-    "@craco/craco": "^6.4.3",
-    "@svgr/webpack": "^5.0.0",
-    "@testing-library/jest-dom": "^5.16.1",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/user-event": "^12.8.3",
-    "@types/jest": "^26.0.24",
-    "@types/node": "^12.20.42",
-    "@types/react": "^17.0.38",
-    "@types/react-dom": "^17.0.11",
-    "concurrently": "^6.5.1",
-    "cross-env": "^7.0.3",
-    "electron": "^13.6.7",
-    "electron-builder": "^23.0.3",
-    "electron-fix": "^1.1.3",
-    "electronmon": "^2.0.2",
-    "jest": "^27.4.7",
-    "react-error-overlay": "6.0.9",
-    "react-scripts": "^4.0.3",
-    "react-test-renderer": "^17.0.2",
-    "ts-loader": "^8.2.0",
-    "typescript": "^4.5.5",
-    "url-loader": "^4.1.1",
-    "wait-on": "^6.0.0",
-    "webpack-cli": "^4.9.2"
-  }
+	"name": "@berty/web",
```

Comment:
Same here, fix the lint and do you remember why you update some libraries in the web package ?